docs/deploying/fly_io.md

```diff
@@ -0,0 +1,148 @@
+# Fly.io
+
+:::caution
+
+This deployment guide is expermiental and may not reliably work each time or for every user.
+We came across issues when deploying to Fly.io, which may render your application not
+behaving as expected.
```

Comment:
Let's add info about our problems i.e. we sometimes needed to wait until UDP traffic being forwarded and unclear documentation in terms of opening socket and mapping ports